repo/packages/M/marathon/0/config.json

```diff
@@ -1,147 +1,181 @@
+        "mesos-authentication-principal": {
+          "description": "The Mesos principal used for authentication.",
+          "type": "string"
+        },
+        "mesos-authentication-secret-file": {
+          "description": "The path to the Mesos secret file containing the authentication secret.",
+          "type": "string"
+        },
+        "mesos-role": {
+          "description": "Mesos role for this framework.",
+          "type": "string"
+        },
+        "reconciliation-initial-delay": {
+          "description": "The delay, in milliseconds, before Marathon begins to periodically perform task reconciliation operations.",
+          "minimum": 0,
+          "type": "integer"
+        },
+        "reconciliation-interval": {
+          "description": "The period, in milliseconds, between task reconciliation operations.",
+          "minimum": 0,
+          "type": "integer"
+        },
+        "ssl-keystore-password": {
+          "description": "Password for the keystore supplied with the marathon/ssl-keystore-path option.",
+          "type": "string"
+        },
+        "ssl-keystore-path": {
+          "description": "Path to the SSL keystore. SSL will be enabled if this option is supplied. The keystore can be downloaded into Marathon's working directory by using the uris property.",
+          "type": "string"
+        },
+        "uris": {
+          "default": [],
+          "description": "List of URIs that will be download and made available in the current working directory of Marathon. For example this can be used to download Java keystore file for SSL configuration.",
+          "items": {
+            "pattern": "^[\\s]+",
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "zk": {
+          "default": "zk://master.mesos:2181/universe",
+          "description": "Parent ZooKeeper URL for storing state. The framework name is joined to the path of this value. Format: zk://host1:port1,host2:port2,.../path",
+          "type": "string"
+        },
+        "zk-max-versions": {
+          "description": "Limit the number of versions stored for one entity.",
+          "minimum": 0,
+          "type": "integer"
+        },
+        "zk-timeout": {
+          "description": "Timeout for ZooKeeper in milliseconds.",
+          "minimum": 0,
+          "type": "integer"
+        }
+      },
+      "required": [
+        "zk",
+        "framework-name"
+      ],
+      "type": "object"
+    },
+    "mesos": {
+      "depscription": "Mesos specific configuration properties",
+      "properties": {
+        "master": {
+          "default": "zk://master.mesos:2181/mesos",
+          "description": "The URL of the Mesos master. The format is a comma-delimited list of hosts like zk://host1:port,host2:port/mesos. If using ZooKeeper, pay particular attention to the leading zk:// and trailing /mesos! If not using ZooKeeper, standard URLs like http://localhost are also acceptable.",
```

Comment:
Drop `http://` in the "no ZK" example.  If not using ZK, then the value can look like this:

- `localhost:5050`
- `10.0.0.5:5050,10.0.0.6:5050`